src/mal/SHCore.cpp

```diff
@@ -2488,6 +2488,33 @@ SHARDS_API __cdecl SHBool shLispEval(void *env, const char *str, SHVar *output)
     return false;
   }
 }
+
+SHARDS_API __cdecl SHBool shLispAddToEnv(void *env, const char *str, SHVar input, bool reference) {
```

Comment:
Can you add the declaration to `SHLisp.h` too

Reply:
Didn't know we had that file :D